doc/build.md

```diff
@@ -2,6 +2,7 @@
 
 ## Required Components
 You will need the following tools and components to build the Basketweave-S (not included in the kit):
+![components](https://imgur.com/C8kzkQa.png)
```

Comment:
If you show the switch puller I would list it below

Reply:
That makes sense, I'll add it

Reply:
But the puller isn't really needed, or?

Reply:
Nope, just put "(Optional)" I guess. But if it is in the picture, I would explain it so total beginners are not confused.

Reply:
Well I had it on hand because I millmaxed mine but yeah it's clearly not requested